soccer-records-model/src/test/java/cz/muni/fi/pa165/team/TeamMatchGoalRepositoryImplTest.java

```diff
@@ -70,22 +74,14 @@ public void testDeleteGoal()
         em.persist(match);
 
         TeamMatchGoal goal = new TeamMatchGoal(scorer, assistant, match, new Date(time));
-        UUID id = goal.getId();
-
         em.persist(goal);
         em.flush();
 
-        TeamMatchGoal dbGoal = em.find(TeamMatchGoal.class, id);
-        Assert.assertNotNull(dbGoal);
-
-        teamMatchGoalDAO.deleteGoal(goal);
-
-        dbGoal = em.find(TeamMatchGoal.class, id);
-        Assert.assertNull(dbGoal);
+        TeamMatchGoal dbGoal = teamMatchGoalRepository.getGoalById(null);
     }
 
-    @Test
-    public void testFindGoalById()
+    @Test(expectedExceptions = GoalNotFoundException.class)
```

Comment:
ty vyjimky bych radeji viděl otestovane jak to udělam v userovi, aby se hlidalo, že to zpusobily ty parametry, ktere ocekavame ze to zpusobi... vis co myslim?